architecture-building-systems/CityEnergyAnalyst
# 3004 continue demand sumulation when overheated

The PR resolves #3004.
An new option is added to bypass the overheating warning.

To test:
- switch to this branch
- delete the local `cea.config` (in my case, it is here: `C:\Users\shsieh`)
- from dashboard, run demand script on projects with overheated buildings
- follow the error message to turn off the overheating warning, and run demand simulation again
- the simulation should be completed after turning off the warning.
